### PR TITLE
interface: add additional print column directives for member cluster API

### PR DESCRIPTION
**Pull request by michaelawyu**
### Description of your changes

This PR adds additional print column directives for the MemberCluster API so that properties such as node count, allocatable CPU and memory will be printed when getting the member clusters via kubectl.

I have:

- [x] Run `make reviewable` to ensure this PR is ready for review.

### How has this code been tested

- [x] Manual testing

### Special notes for your reviewer

**Important**: the print-column directives have been updated to use the new node count property name.

### CODE REVIEW

File: apis/cluster/v1/membercluster_types.go [hunk 1]
```diff
@@ -17,6 +17,9 @@ import (
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:JSONPath=`.status.conditions[?(@.type=="Joined")].status`,name="Joined",type=string
 // +kubebuilder:printcolumn:JSONPath=`.metadata.creationTimestamp`,name="Age",type=date
+// +kubebuilder:printcolumn:JSONPath=`.status.properties.kubernetes-fleet\.io/node-count.value`,name="NodeCount",type=string
+// +kubebuilder:printcolumn:JSONPath=`.status.resourceUsage.allocatable.cpu`,name="AllocatableCPU",type=string
```
Comment (ryanzhang-oss) [inline apis/cluster/v1/membercluster_types.go:21]: I think we need available instead of allocatable

Comment (ryanzhang-oss) [reply apis/cluster/v1/membercluster_types.go:21]: or both

Comment (britaniar) [reply apis/cluster/v1/membercluster_types.go]: Allocatable will be part of -o wide and available part of regular.

**Review by ryanzhang-oss — APPROVED** [review]

**Review by ryanzhang-oss — COMMENTED** [review]
https://secrets-store-csi-driver.sigs.k8s.io/concepts.html

**Review by ryanzhang-oss — APPROVED** [review]

**Pull request merged**